app/controllers/MoreInformationController.scala

```diff
@@ -31,7 +32,7 @@ import javax.inject.{Inject, Singleton}
 class MoreInformationController @Inject() (
   actions: Actions,
   val controllerComponents: MessagesControllerComponents,
-  navigator: Navigator,
```

Comment:
Is it worth pulling the other ticket into this branch so this change can be made? The other PR has been approved already and I just worry that if both get merged this will cause problems. And tbh might have merge conflicts anyway (or this can be done to the other PR)

Reply:
This was meant to be for the above TODO 😬